src/js/App/Sidenav/Navigation.js

```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 import { appNavClick, clearActive } from '../../redux/actions';
 import NavigationItem from './NavigationItem';
 
-const basepath = `${document.baseURI}platform/`;
+const basepath = `${document.baseURI}`;
```

Comment:
This is already addressed in https://github.com/RedHatInsights/insights-chrome/pull/185